src/components/componentStyles/Navbar.modules.css

```diff
@@ -0,0 +1,61 @@
+nav {
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  display: flex;
```

Comment:
- [ ] Kindly make sure that CSS bad practices[(article for support)](https://speckyboy.com/good-bad-css-practices/) like no repeating of the same rule. For example, you have used display: flex 7 times. I would suggest you create a class, for example, "flex-box" and use it at all places that you need on your code.
- [ ] The same should be done with other CSS rules where background-color has been used.